### PR TITLE
Fix sign-in button UI

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

- Move the sign in button to left of export button
- Refactor shortcut toolbar code

DynamoSandbox_Chry1pEjnw

About Box update, added IDSDK disclaimer

Screen Shot 2022-11-21 at 11 58 06 AM

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -18,6 +18,7 @@
     <Compile Remove="Graph\ConnectorPinModel.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="PresentationCore" />
```
Comment (mjkkirschner) [inline src/DynamoCore/DynamoCore.csproj:21]: I don't think this is ok - if I am understanding correctly this binary is WPF and now DynamoCore depends on it. We'll have to find another way. What did you need this for?

**Review by mjkkirschner — CHANGES_REQUESTED** [review]

File: src/DynamoCore/DynamoCore.csproj (hunk 1 above)
Comment (mjkkirschner) [reply src/DynamoCore/DynamoCore.csproj:21]: It seems like maybe this can just be reverted?

Comment (zeusongit) [reply src/DynamoCore/DynamoCore.csproj:21]: yep

File: src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs [hunk 2]
```diff
@@ -34,21 +37,27 @@ public ObservableCollection<ShortcutBarItem> ShortcutBarRightSideItems
...
         }
+        private readonly DynamoModel _viewModel;
+        public ICommand SignOutCommand { get; private set; }
 
         /// <summary>
         /// Construct a ShortcutToolbar.
         /// </summary>
         /// <param name="updateManager"></param>
-        public ShortcutToolbar(IUpdateManager updateManager)
+        public ShortcutToolbar(DynamoModel dm)
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs:47]: I only have concern for this known there will be conflict with @filipeotero 's https://github.com/DynamoDS/Dynamo/pull/13550

File: src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml [hunk 3]
```diff
@@ -150,10 +150,10 @@
                                             <Style TargetType="{x:Type TextBlock}">
                                                 <Setter Property="Text" Value="{x:Static p:Resources.SignInButtonText}" />
                                                 <Style.Triggers>
-                                                    <DataTrigger Binding="{Binding AuthenticationManager.LoginState}" Value="LoggedIn">
-                                                        <Setter Property="Text" Value="{Binding AuthenticationManager.Username}" />
+                                                    <DataTrigger Binding="{Binding LoginState}" Value="LoggedIn">
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml:153]: exellent!

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
One thing I forgot to ask you is update licence.txt as well. In addition, since the release version is different in RC branch, I could not cherry-pick this one OOTB:(

**Pull request merged**